clients/snowflake/writes.go

```diff
@@ -49,7 +50,16 @@ func (s *Store) Merge(tableData *optimization.TableData) error {
 			}
 		}
 
-		err = shared.Merge(s, tableData, s.config, types.MergeOpts{})
+		var additionalEqualityStrings []string
+		if len(tableData.TopicConfig.AdditionalMergeColumns) > 0 {
+			for _, additionalMergeCol := range tableData.TopicConfig.AdditionalMergeColumns {
```

Comment:
Does `additionalMergeCol` need to be escaped?

Reply:
Yeah it does / should. Will add